.circleci/config.base.yml

```diff
@@ -1228,8 +1228,8 @@ commands:
                 command: |
                   source .circleci/local_publish_helpers.sh
                   cd packages/amplify-e2e-tests
-                  retry yarn run e2e --detectOpenHandles --maxWorkers=3 $TEST_SUITE
-                no_output_timeout: 60m
+                  retry yarn run e2e --force-exit --detectOpenHandles --maxWorkers=3 $TEST_SUITE
+                no_output_timeout: 45m
```

Comment:
Is the random [delay](https://github.com/aws-amplify/amplify-cli/pull/11468/files#diff-2ad94d8b77618868ab7947dab3e34b17ddf57cc56529b6d35d39e9050376e520L248) included or excluded in the 45m?

Reply:
excluded; because the console will output immediately after assuming credentials; this is only for tests that hang (no output at all) for 45 minutes, which should be a red flag